lib/send-html.js

```diff
@@ -0,0 +1,35 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ejs = require('ejs');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = sendHTML;
+
+function sendHTML(res, data, options) {
```

Comment:
I personally prefer .NET naming convention, where acronyms are in pascal case - `sendHtml`. We don't have a formal rule for this, feel free to ignore this comment.